src/screens/ProfileScreen/DetailsScreen/ProgressDetailScreen.tsx

```diff
@@ -103,18 +83,13 @@ const styles = StyleSheet.create({
     width: '100%',
     alignItems: 'center',
   },
-  // titleText: {
-  //   fontSize: 24,
-  //   padding: 5,
-  //   color: 'black',
-  // },
   titleContainer: {
     padding: 10,
     alignItems: 'center',
     marginBottom: 12,
   },
   titleText: {
-    color: '#041187', // You can customize the text color
+    color: '#041187',
```

Comment:
use color variable

Reply:
for color variable, we need to add a new issue  as we have to update it everywhere with testing.